docker-files/database/init-files/test_mysql_init.sql

```diff
@@ -13,7 +11,13 @@ CREATE TABLE "auctions" (
 	"duration" INTERVAL NOT NULL,
 	"begin_time" TIMESTAMP NOT NULL,
 	PRIMARY KEY("auction_id")
-);
+)
+INSERT INTO "auctions" ("auction_id","created_at","updated_at","employee_id","duration","begin_time") VALUES
+(07c3ce66-bda2-4e10-3752-8665cd96b26d,2024-11-5-12:00,2024-11-5-12:00,cd6c194b-aecb-8bac-c7cf-ad569ee9d5ea,'30 minutes',2024-11-10-12:00),
```

Comment:
```suggestion
('07c3ce66-bda2-4e10-3752-8665cd96b26d','2024-11-5 12:00','2024-11-5 12:00','cd6c194b-aecb-8bac-c7cf-ad569ee9d5ea',interval'30 minutes','2024-11-10 12:00'),
```
日付の形式とクオーテーションで囲むようにするのと、intervalのところは接頭辞をつけてもらえると！！